### ISSUE TITLE
Hyperparameter-Tuning for an DQN-Agent used for PMSM-control

### ISSUE THREAD

**Issue by GitPascalP** [upb-lea/gym-electric-motor#99]
Tuning Parameters for the basic DQN-Implementation.

Parameters to be tuned can be:

- exploration of the agent
- batch size, update interval
- learning rate from optimizer
- ...

Possible tuning Algorithms:
- basic grid/random-search
- bayesian-optimization
- ...

**Comment by wkirgsn** [comment]
obsolete working package
